test/test_genomicsdb.py

```diff
@@ -10,160 +11,243 @@
 from genomicsdb.protobuf import genomicsdb_export_config_pb2 as query_pb
 from genomicsdb.protobuf import genomicsdb_coordinates_pb2 as query_coords
 
+
 @pytest.fixture()
 def setup():
-  tmp_dir = tempfile.TemporaryDirectory().name
-  print("tmp_dir")
-  if not os.path.exists(tmp_dir):
-    os.makedirs(tmp_dir)
-  else:
-    sys.exit("Aborting as temporary directory seems to exist!")
-  tar = tarfile.open("test/inputs/sanity.test.tgz")
-  tar.extractall(tmp_dir)
-  os.chdir(tmp_dir)
-  yield
-  shutil.rmtree(tmp_dir)
+    tmp_dir = tempfile.TemporaryDirectory().name
```

Comment:
Sorry, this got reformatted automatically by Emacs. I can bring it back to where it was before if it is too difficult to read. But the arrow changes are from L108-L131!